main/browser-config.ts

```diff
@@ -69,9 +69,10 @@ export default class BrowserConfig {
         if (this.window_state === null) {
             return null;
         }
-        win.once('closed', () => {
+        win.on('resize', () => {
```

Comment:
Oh, I also understood the root cause.  

I think `'close'` event is more suitable to save window state ([document](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#event-close)).  `close` event is emitted before window is unloaded.  If we use `resize` event, every window resize will save window state.